lib/active_merchant/billing/gateways/nmi.rb

```diff
@@ -102,6 +106,7 @@ def store(payment_method, options = {})
         add_customer_data(post, options)
         add_vendor_data(post, options)
         add_merchant_defined_fields(post, options)
+        add_level3_fields(post, options)
```

Comment:
Seems odd that we would pass this info for `store` 🤷‍♂

Reply:
I agree I don't think this is necessary.

Reply:
👍Thanks -- I removed the fields from ```store```.